NewBank/newbank/server/Customer.java

```diff
@@ -10,6 +10,7 @@
 
     private ArrayList<Account> accounts;
     private LinkedList<Transaction> transactionHistory = new LinkedList<>();
+    private ArrayList<Microloan> microloans = new ArrayList<>();
```

Comment:
This field is never used and could be deleted. But don't worry, we will do that with Eamonn to avoid merge conflicts